descriptor/cip/src/main/java/org/openscience/cdk/geometry/cip/rules/MassNumberRule.java

```diff
@@ -63,7 +64,9 @@ private Integer getMassNumber(ILigand ligand) {
         Integer massNumber = ligand.getLigandAtom().getMassNumber();
         if (massNumber != null) return massNumber;
         if (factory == null) return 0;
-        return factory.getMajorIsotope(ligand.getLigandAtom().getSymbol()).getMassNumber();
+        IIsotope isotope = factory.getMajorIsotope(ligand.getLigandAtom().getSymbol());
+        if (isotope == null) return 0;
```

Comment:
Ok, note Centres does a much better job of CIP labels. https://github.com/SiMolecule/centres